packages/mc-html-template/lib/utils/replace-html-placeholders.js

```diff
@@ -14,8 +14,10 @@ const replaceHtmlPlaceholders = (indexHtmlContent, config) =>
   indexHtmlContent
     .replace(
       new RegExp('__CDN_URL__', 'g'),
-      // Ensure there is a trailing slash
-      `${config.cdnUrl.replace(/\/$/, '')}/`
+      config.cdnUrl
```

Comment:
In case the `cdnUrl` value is an empty string, we shouldn't append a trailing slash